python/lsst/ts/envsensors/command_handler.py

```diff
@@ -317,6 +342,9 @@ def _get_device(self, device_configuration: dict) -> BaseDevice:
         if self.simulation_mode == 1:
             from .device import MockDevice
 
+            self.log.info(
```

Comment:
Please consider debug level here and at line 360 376, etc (look through the log messages)

Reply:
Done